lib/oppen/printer.rb

```diff
@@ -175,10 +180,16 @@ def handle_end(token)
     # @see Token::Break
     def handle_break(token)
       if scan_stack.empty?
+        tokens[-1] = nil
         @left = 0
         @left_total = 1
         @right = 0
         @right_total = 1
+
+        if @last_string_type == :whitespace
+          print_stack.erase
+          @last_string_type = nil
+        end
```

Comment:
If I wasn't clear, it's my mistake, but I'd like to have a config flag that says whether we want to trim trailing whitespaces or not since it's outside the classic Oppen algorithm.

It might make the code look dirty, but we might find a way to cleanly factor out those "strategies" … but let's attack things one at a time.